examples/histogram.py

```diff
@@ -84,10 +85,88 @@ def histogram(neurons, feature, new_fig=True, subplot=False, **kwargs):
 
     feature_values = [getattr(neu, 'get_' + feature)() for neu in neurons]
 
-    neu_labels = ['neuron_id' for neu in neurons]
+    neu_labels = [neu.name for neu in neurons]
 
-    ax.hist(feature_values, bins=bins, cumulative=cumulative, label=neu_labels)
+    ax.hist(feature_values, bins=bins, cumulative=cumulative, label=neu_labels, normed=normed)
 
     kwargs['no_legend'] = len(neu_labels) == 1
 
     return common.plot_style(fig=fig, ax=ax, **kwargs)
+
+
+def population_feature_values(pops, feature):
+    '''Extracts feature values per population
+    '''
+    pops_feature_values = []
+
+    for pop in pops:
+
+        feature_values = [getattr(neu, 'get_' + feature)() for neu in pop.neurons]
+
+        # ugly hack to chain in case of list of lists
+        if any([isinstance(p, (list, np.ndarray)) for p in feature_values]):
+
+            feature_values = list(chain(*feature_values))
+
+        pops_feature_values.append(feature_values)
+
+    return pops_feature_values
+
+
+def population_histogram(pops, feature, new_fig=True, normed=False, subplot=False, **kwargs):
+    '''
+    Plot a histogram of the selected feature for the population of neurons.
+    Plots x-axis versus y-axis on a scatter|histogram|binned values plot.
+
+    More information about the plot and how it works.
+
+    Parameters
+
+        populations : populations list
+
+        feature : str
+        The feature of interest.
+
+    Options
+
+        bins : int
+        Number of bins for the histogram.
+
+        cumulative : bool
+        Sets cumulative histogram on.
+
+        subplot : bool
+            Default is False, which returns a matplotlib figure object. If True,
+            returns a matplotlib axis object, for use as a subplot.
+
+    Returns
```

Comment:
Parameters, Options and Returns will need `:` in order for the docs to be build correctly. I am not sure if "Options" will work either.